README.md

```diff
@@ -77,14 +77,28 @@ Place it into project library path in your IDE.
 ## Usage
 
 1. [Class Tarantool](#class-tarantool)
-1. [Predefined Constants](#predefined-constants)
+2. [Predefined Constants](#predefined-constants)
+3. [Manipulation connection](#manipulation-connection)
+  * [connect](#connect)
+  * [disconnect, close](#disconnect-close)
+  * [authenticate](#authenticate)
+  * [flushSchema, flush_schema](#flushschema-flush_schema)
+  * [ping](#ping)
+4. [Database queries](#database-queries)
+  * [select](#select)
+  * [insert, replace](#insert-replace)
+  * [call](#call)
+  * [eval, evaluate](#eval-evaluate)
+  * [delete](#delete)
+  * [update](#update)
+  * [upsert](#upsert)
 
 ### Class Tarantool
 
 _**Description**_: Creates a Tarantool client
 
 ``` php
-tarantool_object = new Tarantool([host = 'localhost'[, port = 3301]])
+$tarantool_object = new Tarantool([$host = 'localhost'[, $port = 3301]])
```

Comment:
I would use `<visibility> <ClassName::menthodName>` format to describe method signatures, e.g.:

```php
public Tarantool::__construct([$host = 'localhost'[, $port = 3301]])
```

See, for example, how it's done for PDO constructor in the PHP manual: http://php.net/manual/en/pdo.construct.php.